scripts/demo_example.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+from __future__ import print_function
```

Comment:
Just as a matter of consistency / style, you should probably include the same header lines that we do on all of our python files:
```
from __future__ import division
from __future__ import print_function
from __future__ import unicode_literals
```